<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Gui</class>
    <widget class="QMainWindow" name="Gui">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1308</width>
                <height>680</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Gui</string>
        </property>
        <widget class="QWidget" name="centralWidget">
            <widget class="QGroupBox" name="groupBox">
                <property name="geometry">
                    <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>331</width>
                        <height>611</height>
                    </rect>
                </property>
                <property name="title">
                    <string>Interface 1</string>
                </property>
                <property name="alignment">
                    <set>Qt::AlignCenter</set>
                </property>
                <widget class="QComboBox" name="comboBox">
                    <property name="geometry">
                        <rect>
                            <x>100</x>
                            <y>70</y>
                            <width>201</width>
                            <height>23</height>
                        </rect>
                    </property>
                </widget>
                <widget class="QLineEdit" name="lineEdit">
                    <property name="geometry">
                        <rect>
                            <x>100</x>
                            <y>100</y>
                            <width>201</width>
                            <height>23</height>
                        </rect>
                    </property>
                </widget>
                <widget class="QLineEdit" name="lineEdit_2">
                    <property name="geometry">
                        <rect>
                            <x>100</x>
                            <y>130</y>
                            <width>201</width>
                            <height>23</height>
                        </rect>
                    </property>
                </widget>
                <widget class="QLabel" name="label">
                    <property name="geometry">
                        <rect>
                            <x>20</x>
                            <y>70</y>
                            <width>67</width>
                            <height>17</height>
                        </rect>
                    </property>
                    <property name="text">
                        <string>Interface</string>
                    </property>
                </widget>
                <widget class="QLabel" name="label_2">
                    <property name="geometry">
                        <rect>
                            <x>20</x>
                            <y>100</y>
                            <width>71</width>
                            <height>20</height>
                        </rect>
                    </property>
                    <property name="text">
                        <string>IP Address</string>
                    </property>
                </widget>
                <widget class="QLabel" name="label_3">
                    <property name="geometry">
                        <rect>
                            <x>20</x>
                            <y>130</y>
                            <width>67</width>
                            <height>17</height>
                        </rect>
                    </property>
                    <property name="text">
                        <string>IP Mask</string>
                    </property>
                </widget>
                <widget class="QPushButton" name="pushButton">
                    <property name="geometry">
                        <rect>
                            <x>140</x>
                            <y>170</y>
                            <width>91</width>
                            <height>27</height>
                        </rect>
                    </property>
                    <property name="text">
                        <string>Set</string>
                    </property>
                </widget>
            </widget>
            <widget class="QGroupBox" name="groupBox_2">
                <property name="geometry">
                    <rect>
                        <x>330</x>
                        <y>0</y>
                        <width>331</width>
                        <height>611</height>
                    </rect>
                </property>
                <property name="title">
                    <string>Interface 2</string>
                </property>
                <property name="alignment">
                    <set>Qt::AlignCenter</set>
                </property>
                <widget class="QComboBox" name="comboBox_2">
                    <property name="geometry">
                        <rect>
                            <x>100</x>
                            <y>70</y>
                            <width>201</width>
                            <height>23</height>
                        </rect>
                    </property>
                </widget>
                <widget class="QLineEdit" name="lineEdit_3">
                    <property name="geometry">
                        <rect>
                            <x>100</x>
                            <y>100</y>
                            <width>201</width>
                            <height>23</height>
                        </rect>
                    </property>
                </widget>
                <widget class="QLineEdit" name="lineEdit_4">
                    <property name="geometry">
                        <rect>
                            <x>100</x>
                            <y>130</y>
                            <width>201</width>
                            <height>23</height>
                        </rect>
                    </property>
                </widget>
                <widget class="QLabel" name="label_4">
                    <property name="geometry">
                        <rect>
                            <x>20</x>
                            <y>70</y>
                            <width>67</width>
                            <height>17</height>
                        </rect>
                    </property>
                    <property name="text">
                        <string>Interface</string>
                    </property>
                </widget>
                <widget class="QLabel" name="label_5">
                    <property name="geometry">
                        <rect>
                            <x>20</x>
                            <y>100</y>
                            <width>71</width>
                            <height>20</height>
                        </rect>
                    </property>
                    <property name="text">
                        <string>IP Address</string>
                    </property>
                </widget>
                <widget class="QLabel" name="label_6">
                    <property name="geometry">
                        <rect>
                            <x>20</x>
                            <y>130</y>
                            <width>67</width>
                            <height>17</height>
                        </rect>
                    </property>
                    <property name="text">
                        <string>IP Mask</string>
                    </property>
                </widget>
                <widget class="QPushButton" name="pushButton_2">
                    <property name="geometry">
                        <rect>
                            <x>150</x>
                            <y>170</y>
                            <width>91</width>
                            <height>27</height>
                        </rect>
                    </property>
                    <property name="text">
                        <string>Set</string>
                    </property>
                </widget>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menuBar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>1308</width>
                    <height>27</height>
                </rect>
            </property>
        </widget>
        <widget class="QToolBar" name="mainToolBar">
            <attribute name="toolBarArea">
                <enum>TopToolBarArea</enum>
            </attribute>
            <attribute name="toolBarBreak">
                <bool>false</bool>
            </attribute>
        </widget>
        <widget class="QStatusBar" name="statusBar"/>
    </widget>
    <layoutdefault spacing="6" margin="11"/>
    <resources/>
    <connections/>
</ui>
